src/System/Wlog/Handler.hs

```diff
@@ -31,7 +31,10 @@ import           System.Wlog.Severity  (LogRecord, Severity)
 import           Universum
 
 -- | Tag identifying handlers.
-data LogHandlerTag = HandlerFilelike FilePath | HandlerOther String
+data LogHandlerTag
```

Comment:
Could you clarify more in documentation what is the difference between `HandlerFileLike` and `HandlerOther`?